tests/backup/backup_upgrade_test.go

```diff
@@ -642,7 +650,8 @@ var _ = Describe("{PXBackupEndToEndBackupAndRestoreWithUpgrade}", func() {
 					restoreName := fmt.Sprintf("%s-%s", config.namePrefix, RandomString(4))
 					log.InfoD("Restoring next single namespace schedule backup [%s] in cluster [%s] with restore [%s] and namespace mapping %v", nextScheduleBackupName, destinationClusterName, restoreName, config.namespaceMapping)
 					if config.replacePolicy == ReplacePolicy_Retain {
-						err = CreateRestore(restoreName, nextScheduleBackupName, config.namespaceMapping, destinationClusterName, orgID, ctx, config.storageClassMapping)
+						appContextsToBackup := FilterAppContextsByNamespace(srcClusterContexts, []string{namespace})
+						err = CreateRestoreWithValidation(ctx, restoreName, nextScheduleBackupName, config.namespaceMapping, config.storageClassMapping, destinationClusterName, orgID, appContextsToBackup)
```

Comment:
I do not think we can run `CreateRestoreWithValidation` in parallel. I apologize. Can you please replace the Parallel parameter with Sequential in TaskHandler if it processes `CreateRestoreWithValidation`? I also hope that this test case does not take a backup from the destination cluster.

Reply:
done